dts/msm8916/msm8939-xiaomi-ido.dts

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+
+/ {
+	qcom,msm-id = <0xef 0x30000 0xf1 0x30000 0x107 0x30000>;
+	qcom,board-id = <0x1000b 0xb 0x2000b 0xb>;
+
+	model = "Qualcomm Technologies, Inc. MSM8939v3.0 QRD SKUK";	
+	compatible = "qcom,msm8939-qrd-ido", "qcom,msm8939-qrd", "qcom,msm8939", "qcom,qrd", "lk2nd,device";
+
+
+	panel {
+		qcom,mdss_dsi_hx8394f_boe_720p_video {
```

Comment:
Is this really the only panel supported? If so, you should change it to matching cmdline.